src/test/java/picocli/ArgGroupTest.java

```diff
@@ -4071,6 +4072,20 @@ public void testIssue1384() {
         assertEquals(obj.argGroup.param2, "b");
     }
 
+
+    /**
+     * Tests issue 1409 https://github.com/remkop/picocli/issues/1409
+     * This specific test supplies values to the group one values, leaving
+     * the group two values uninitialized.
+     * <p>
+     * The test verifies that x, 1A, 1B, 2A, and 2B values are correct after
+     * building the command using CommandLine.java and parsing the arguments.
+     * @author remkop, madfoal
+     * @version 1
+     * @params null
+     * @returns null
+     *
```

Comment:
Can we remove the `@version 1`, `@params null`, `@returns null` and the empty line at the bottom of the javadoc?